package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@hospitalrun-org/server",
+  "version": "1.0.0",
+  "description": "Fastify Application",
+  "main": "app.ts",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "ts-node ./node_modules/tap/bin/run.js test/**/*.test.ts",
```

Comment:
this lines will not work on Windows. 
Also the glob paths have to be surrounded by \"\". Again, no single quotes because of Windows.